.github/workflows/tests.yml

```diff
@@ -3,30 +3,43 @@ name: Build tests
 on: [push, pull_request]
 
 env:
-  SELINUX_USERSPACE_VERSION: checkpolicy-3.1
+  # 3.4 fails to validate, fixed with 88a703399f3f
+  # ("libsepol: fix validation of user declarations in modules")
+  SELINUX_USERSPACE_VERSION: checkpolicy-3.3
```

Comment:
Does selint require the new userspace? I want to keep this at refpolicy's minimum supported userspace version.